### ISSUE TITLE
Being able to embed icons into a text layer

### ISSUE THREAD

**Issue by riverole** [paperize/paperize#15]
**Is your feature request related to a problem? Please describe.**
I find that it's very useful to embed svg icons to layers of text.

**Describe the solution you'd like**
It would be great to be able to embed icons in a text. In the text you could have some specific magic combination, for example :icon_1:, which then would be substituted with a specific icon. Of course, you would need to define those icons before.

**Describe alternatives you've considered**
Basically having a separate layer for text and icons (images). However, that requires having to redesign all my cards.
